doc/source/whatsnew/v1.4.0.rst

```diff
@@ -20,7 +20,8 @@ Enhancements
 Improved warning messages
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Previously, warning messages may have pointed to lines within the pandas library. Running the script ``setting_with_copy_warning.py``
+Previously, warning messages may have pointed to lines within the pandas
+library. Running the script ``setting_with_copy_warning.py``
```

Comment:
this change and some others were just re-wrapping. Next time i'll do this separately (or separate commit) to make review easier.